code-challenges.js

```diff
@@ -8,47 +8,84 @@
 
 // Pseudo code:
 
-// Set one:
-const fruit1 = "apple"
-const fruit2 = "banana"
-// Expected outcome: "banana"
-
-// Set two:
-const fruit3 = "cherry"
-const fruit4 = "kiwi"
+// // Set one:
+// const fruit1 = "apple"
+// const fruit2 = "banana"
+// // Expected outcome: "banana"
+
+// // Set two:
+// const fruit3 = "cherry"
+// const fruit4 = "kiwi"
 // Expected outcome: "cherry"
 
 // --------------------1) Create a function that takes in a number and determines if the given number is below boiling point, at boiling point, or above boiling point. Boiling point is 212 degrees Fahrenheit. Use the test variables provided below.
 
 // Pseudo code:
-
-const temperature1 = 42
-// Expected output: "42 is below boiling point"
-
-const temperature2 = 350
-// Expected output: "350 is above boiling point"
-
-const temperature3 = 212
-// Expected output: "212 is at boiling point"
+// create a variable for boilingPoint that cant be changed, so const.
+// create a function and insert a parameter for a inserted number.
+// create a condition using if/else statements
+// return the num with a string interpolation
+// anything other than a number will return a error message
+
+// const temperature1 = 42
+// // Expected output: "42 is below boiling point"
+
+// const temperature2 = 350
+// // Expected output: "350 is above boiling point"
+
+// const temperature3 = 212
+// // Expected output: "212 is at boiling point"
+
+        const boilingPoint = 212
+        const temp = (num) => {
+            if (num > boilingPoint) {
+                return `${num} is above boiling point` 
+            }
+            else if (num === boilingPoint) {
+                return `${num} is at boiling point`
+            }
+            else if (num < boilingPoint) {
+                return `${num} is below boiling point`
+            }
+            else {
+                return 'error'
+            }
+        }
+        console.log(temp())
```

Comment:
Great function and pseudo code. Quick note, when you create a function with a parameter, it will expect an argument when you invoke the function. In this case, you will need to pass a number in on line 54.